### PR TITLE
[PVR] Fix timeshift gui info. Must be reset if nothing is playing anymore

### PR DESCRIPTION
**Pull request by ksooo**
Happy new year. This is my first PR in 2018. ;-)

The bug led to timeshift gui controls (buffer start value, ...) appearing when not timeshifting anymore.

The fix was runtime tested on macOS, latest master.

@Jalle19 mind taking a look?

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by rickychester** [comment]
I like this app kodi but I'm trying add it on two you tube and would to sale some art work of mine the future for marketing a art business soon so I know this a great app for that protect  sincerely ricky

File: xbmc/pvr/PVRGUIInfo.cpp [hunk 1]
```diff
@@ -246,8 +247,20 @@ void CPVRGUIInfo::UpdateTimeshift(void)
 {
   if (!CServiceBroker::GetPVRManager().IsPlayingTV() && !CServiceBroker::GetPVRManager().IsPlayingRadio())
   {
+    // If nothing is playing (anymore), there is no need to poll the timeshift values from the clients.
     CSingleLock lock(m_critSection);
-    m_iStartTime = 0;
+    if (m_bHasTimeshiftData)
```
Comment (Jalle19) [inline xbmc/pvr/PVRGUIInfo.cpp]: Shouldn't this be negated?

Comment (ksooo) [reply xbmc/pvr/PVRGUIInfo.cpp]: No. If not pledging and there is data out must be cleared. All good imo.

**Pull request merged**